salt/elife-xpub/init.sls

```diff
@@ -41,6 +41,11 @@ elife-xpub-environment-variables-for-configuration:
             export PUBSWEET_SECRET={{ pillar.elife_xpub.pubsweet.secret }}
             export S3_BUCKET={{ pillar.elife_xpub.s3.bucket }}
             export NODE_CONFIG_ENV={{ pillar.elife_xpub.deployment_target }}
+            export MECA_SFTP_HOST={{ pillar.elife_xpub.meca.sftp.connectionOptions.host }}
```

Comment:
some dummy values should be added to https://github.com/elifesciences/elife-xpub-formula/blob/master/salt/pillar/elife-xpub.sls as these will be copied into the AWS default

Reply:
`connectionOptions` goes against Salt convention to use snake case for pillar names `connection_options`; I don't think it's forbidden but non-standard

Reply:
Ok ... happy to change this :+1: